### PR TITLE
added Disboard.org

### PR DESCRIPTION
**Pull request by Zenternal**

### CODE REVIEW

**Comment by frozaum** [comment]
Hello, as of right now, we won't be able to review your pull request due to the temporarily locking of this repository that we announced in #1946, so we gladly request your patience until everything is sorted out. :)

**Comment by Bas950** [comment]
Unlocked (No longer on hold) - Please update your fork to get your Pull Request reviewed, if you don't do it within the next 3 days the Pull Request will be closed.

File: websites/D/Disboard/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    "id": "165170547192233985"
+  },
+  "service": "Disboard",
+  "description": {
+    "en": "DISBOARD is the public Discord server listing community"
+  },
+  "url": "disboard.org",
+  "version": "1.0.0",
+  "logo": "https://i.imgur.com/d7ko9rn.png",
+  "thumbnail": "https://i.imgur.com/d7ko9rn.png",
```
Comment (frozaum) [inline websites/D/Disboard/dist/metadata.json:13]: Please change the thumbnail to a promotional image or a screenshot of the website.

**Comment by doomlerd** [comment]
Don't forget to update your fork

**Pull request closed without merging**